### PR TITLE
FIX #30010 : Use conf TICKET_MESSAGE_MAIL_INTRO instead of translation key

### LINKED ISSUE #30010: When sending ticket by mail, TicketMessageMailIntroText is forced in the message
### Bug

When you notify a user by mail when adding a new message in ticket, the message sent has the translation of TicketMessageMailIntroText on top of your message everytime. It doesn't take the const TICKET_MESSAGE_MAIL_INTRO into account.

### Dolibarr Version

17 - develop

### Environment PHP

_No response_

### Environment Database

_No response_

### Steps to reproduce the behavior and expected behavior

_No response_

### Attached files

_No response_

### PR DESCRIPTION
**Pull request by evarisk-micka**

### CODE REVIEW

File: htdocs/ticket/class/ticket.class.php [hunk 1]
```diff
@@ -2598,7 +2598,7 @@ public function newMessage($user, &$action, $private = 1, $public_area = 0)
 							$subject = '['.$label_title.'- ticket #'.$object->track_id.'] '.$langs->trans('TicketNewMessage');
 
 							// Message send
-							$message = $langs->trans('TicketMessageMailIntroText');
+							$message = getDolGlobalString('TICKET_MESSAGE_MAIL_INTRO');
```
Comment (eldy) [inline htdocs/ticket/class/ticket.class.php:2601]: What if TICKET_MESSAGE_MAIL_INTRO is not configured/empty ?
May be we can use
`$message = getDolGlobalString('TICKET_MESSAGE_MAIL_INTRO', $langs->trans('TicketMessageMailIntroText'));`

Comment (evarisk-micka) [reply htdocs/ticket/class/ticket.class.php]: I agree and it's done. If someone really wants the intro message to disappear he can always input a space character or something similar.

**Review by rycks — APPROVED** [review]
ok for me

**Review by lvessiller-opendsi — APPROVED** [review]

**Comment by lvessiller-opendsi** [comment]
ok for me @eldy

**Pull request merged**